tools/pbxproj_generators/BUILD

```diff
@@ -0,0 +1,28 @@
+load("//xcodeproj:defs.bzl", "xcodeproj")
+load(
+    ":xcodeproj_targets.bzl",
+    "SCHEME_AUTOGENERATION_MODE",
+    "TOP_LEVEL_TARGETS",
+    "XCODE_CONFIGURATIONS",
+    "get_xcode_schemes",
+)
```

Comment:
I was using this pattern still, because I thought I would add fixtures for these... but I don't think we will be using fixtures by the time all of this is done. So I'll probably move those values into this file.